### PR TITLE
Fix/import temp tables cleanup

### PR DESCRIPTION
**Pull request by joelhanson**
Pull request branched off of staging with cherry-picked commits for importing to temp tables

### CODE REVIEW

**Comment by joelhanson** [comment]
Replaces #82

File: talentmap_api/integrations/models.py [hunk 1]
```diff
@@ -140,6 +155,13 @@ def synchronize(self, soap_function="IPMSDataWebService", test=False):
                         break
                     data_elapsed_time = (datetime.datetime.now() - pre_data_time).total_seconds()
                     logger.info(f"Retrieved SOAP response in {data_elapsed_time} seconds")
+
+                    # Save response_xml into temp tables
+                    xml_tree = ET.fromstring(response_xml)
+                    xml_dict = xml_etree_to_dict(xml_tree)
+                    importIn = ImportModel(import_data=xml_dict, source_endpoint=soap_function_name, imported=False)
```
Comment (rtirserio) [inline talentmap_api/integrations/models.py]: `soap_function_name` is always the same thing. Use `self.talentmap_model`

Comment (joelhanson) [reply talentmap_api/integrations/models.py]: Updated and switched base branch to staging

**Comment by rtirserio** [comment]
@joelhanson there is stuff in here related to the homepage management that we dont want

**Comment by joelhanson** [comment]
@rtirserio not sure how those got into here. Probably sloppy merging on my part. Fixed.

**Pull request merged**